src/it/scala/nl/wbaa/testkit/AwaitAtMostTrait.scala

```diff
@@ -0,0 +1,18 @@
+package nl.wbaa.testkit
+
+import scala.concurrent.duration._
+import scala.util.Try
+
+
+trait AwaitAtMostTrait {
+  private val maxWaitTimeForContainers: Int = 10 * 60
+
+  lazy val waitAtMostDuration: FiniteDuration = {
+    val secondsTimeout = sys.env.get("INTEGRATION_TEST_TIMEOUT") match {
+      case Some(timeout) => Try(timeout.toInt).toOption.getOrElse(maxWaitTimeForContainers)
+      case None          => maxWaitTimeForContainers
+    }
```

Comment:
This can be done with a `.flatMap` and yield an `Option[FiniteDuration]`, with the last step being `_.getOrElse(<DEFAULT VALUE>)`